frb/data/Galaxies/121102/FRB121102_host.json

```diff
@@ -3,13 +3,24 @@
     "cosmo": "Planck15",
     "dec": 33.14799784235409,
     "derived": {
-        "M_r": -17.0,
-        "M_r_err": 0.2,
-        "Mstar": 55000000.0,
-        "Mstar_err": 15000000.0,
-        "SFR_nebular": 0.23,
```

Comment:
Do we want to keep SFR_nebular? At least we need to be sure that it’s correctly parsed from the H-alpha flux so we still have the measurement.

Reply:
yeah, it's ok.